pyslackersweb/tasks.py

```diff
@@ -103,8 +116,62 @@ def sync_slack_timezones(app: web.Application):
                 slack_timezones=dict(counter.most_common(100)),
                 slack_user_count=sum(counter.values()),
             )
-        except Exception:
+        except Exception:  # pylint: disable=broad-except
             logger.exception("Error refreshing slack user cache")
-            raise
+            return
+
+    return _sync_slack_users
+
+
+def sync_slack_channels(app: web.Application):
+    session = app["client_session"]
+
+    async def _sync_slack_channel():
+        logger.debug("Refreshing slack channels cache.")
+        oauth_token = app["slack_token"]
+
+        if oauth_token is None:
+            logger.error("No slack oauth token set, unable to sync slack channels.")
+            return
+
+        try:
+            channels = []
+            while True:
+                params = {}
+                async with session.get(
```

Comment:
We should probably add a `pyslackersweb/util/slack.py` utility file that does the paging and just yields the pages to us to DRY this up a little. 

At some point I'd like to pull the actual HTTP logic out of the tasks, this may be a good time to do it for the slack stuff.

Reply:
Can't DRY it more than that.
I took a quick look at the slack client v2 but it would still requires a bunch of code around compared to slack-sansio